src/ctr_exit.c

```diff
@@ -133,32 +134,13 @@ void container_exit_cb(G_GNUC_UNUSED GPid pid, int status, G_GNUC_UNUSED gpointe
 	g_main_loop_quit(main_loop);
 }
 
-static void close_fd(int *fd)
-{
-	if (*fd >= 0)
-		close(*fd);
-	*fd = -1;
-}
-
 void do_exit_command()
 {
-	if (sync_pipe_fd > 0) {
-		close(sync_pipe_fd);
-		sync_pipe_fd = -1;
-	}
-
 	if (signal(SIGCHLD, SIG_DFL) == SIG_ERR) {
 		_pexit("Failed to reset signal for SIGCHLD");
 	}
 
-	close_logging_fds();
-	close_fd(&terminal_ctrl_fd);
-	close_fd(&winsz_fd_w);
-	close_fd(&winsz_fd_r);
-	close_fd(&mainfd_stdin);
-	close_fd(&attach_socket_fd);
-	close_fd(&console_socket_fd);
-	close_fd(&attach_pipe_fd);
+	close_all_fds_ge_than(3);
```

Comment:
Might be nice to have an inline comment on where '3' came from

Reply:
I thought it was clear they are just the std streams.  I'll add a comment